include/clap/ext/params.h

```diff
@@ -129,13 +129,13 @@ enum {
    // Does this parameter support per note automations?
```

Comment:
We should get note id comment also. Thanks!

Reply:
done.